src/Router/buildClientApp.tsx

```diff
@@ -20,9 +20,14 @@ export function buildClientApp(config: AppConfig): Promise<ClientResolveProps> {
 
       const relayEnvironment = createEnvironment({
         cache: relayBootstrap,
-        user: (sd as any).CURRENT_USER, // FIXME: https://github.com/artsy/reaction/pull/704#pullrequestreview-12508005
+        user: {
+          id: process.env.USER_ID,
+          accessToken: process.env.USER_ACCESS_TOKEN,
```

Comment:
This might blow up in force, will see

Reply:
Yea, what's all this?

Reply:
its the values from our [env file and / or staging](https://github.com/artsy/reaction/blob/master/.env.oss#L9-L10) env. Will need to iterate a bit but this will get us what we need for now.

Reply:
Oh yea I mean, I guess what I was asking is, this is just some intermediate step to have something working, with the intent to circle back and fix up?

Reply:
Yah from force we need to pass a user in. I’ll take care of that tomorrow. This is to unblock you with follow stuff